letsencrypt/client/tests/auth_handler_test.py

```diff
@@ -405,16 +454,25 @@ def test_not_satisfied(self):
         self.handler.paths[dom[3]] = [0]
         self.handler.responses[dom[3]] = ["null"]
 
-        for i in range(4):
+        for i in xrange(4):
             self.assertFalse(self.handler._path_satisfied(dom[i]))
 
 
-def gen_auth_resp(chall_list):  # pylint: disable=missing-docstring
+def gen_auth_resp(chall_list):
+    """Generate a dummy authorization response."""
     return ["%s%s" % (type(chall).__name__, chall.domain)
             for chall in chall_list]
 
 
-def gen_path(str_list, challenges):  # pylint: disable=missing-docstring
+def gen_path(str_list, challenges):
+    """Generate a path for challenge messages
+
+    :param list str_list: list of str, challenge message types
```

Comment:
I recently try to reword docs into something along these lines:
```
:param list str_list: challenge message types (:class:`str`)
```
as container type is already mentioned in `:param list str_list:`